### PR TITLE
Url checker

### PR DESCRIPTION
**Pull request by mqharris**

### CODE REVIEW

File: R/recall_data.R [hunk 1]
```diff
@@ -129,7 +129,7 @@ recall_by_make <- function(make, manufacturer = FALSE,
 #' @examples
 #' \dontrun{
 #' recall_by_model("civic")
-#' recall_by_model(c("Subaru", "Toyota"), start_year = 2008, partial = FALSE)
+#' recall_by_model(c("ranger", "brz"), start_year = 2008, partial = TRUE)
 #' API_KEY <- "xxxxxxxxxxx"
```
Comment (WraySmith) [inline R/recall_data.R:132]: why did the above line change? `partial = TRUE` is the default behavior so doesn't really make sense to have an example with `partial = TRUE`

File: tests/testthat/test-utils.R [hunk 2]
```diff
@@ -40,3 +40,18 @@ test_that("Clean API returns formatted dataframe", {
...
+  url_with_bad_limit <- "https://vrdb-tc-apicast-production.api.canada.ca/eng/vehicle-recall-database/v1/recall/model-name/911|ranger/year-range/2008-2100?limit=-10"
+  expect_error(check_url(url_with_bad_limit))
+
+  url_with_proper_format <- "https://vrdb-tc-apicast-production.api.canada.ca/eng/vehicle-recall-database/v1/recall/make-name/Maz/year-range/1995-2000/count"
+  check_url(url_with_proper_format)
+  url_with_limit <- "https://vrdb-tc-apicast-production.api.canada.ca/eng/vehicle-recall-database/v1/recall/make-name/Maz/year-range/1995-2000/count?limit=10"
+  check_url(url_with_limit)
+  url_with_null_limit <- "https://vrdb-tc-apicast-production.api.canada.ca/eng/vehicle-recall-database/v1/recall/make-name/Maz/year-range/1995-2000/count?limit="
+  check_url(url_with_null_limit)
+})
```
Comment (WraySmith) [inline tests/testthat/test-utils.R:59]: what's the purpose of the null limit check? the API works with this condition - I think this is maybe why the check failed - should just check if its <=0, some of the functions depend on 'limit=' in the url

Comment (WraySmith) [reply tests/testthat/test-utils.R:59]: i see, the null check isn't testing for failure; however, a bunch of the tests aren't wrapped in an `expect_*()` statement

Comment (WraySmith) [reply tests/testthat/test-utils.R:59]: probably just `expect_equal(check_url(url_with_null_limit), url_with_null_limit)` etc

File: R/recall_data.R (hunk 1 above)
Comment (mqharris) [reply R/recall_data.R:132]: Subaru and Toyota aren't models

I changed partial back to FALSE

File: tests/testthat/test-utils.R (hunk 2 above)
Comment (mqharris) [reply tests/testthat/test-utils.R:59]: okay

**Review by WraySmith — APPROVED** [review]

**Pull request merged**